src/components/DecisioningEngine/constants.js

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+export const CONTEXT_KEY = {
+  TYPE: "~type",
+  SOURCE: "~source"
+};
+
+export const MOBILE_EVENT_TYPE = {
```

Comment:
is this related to Mobile? if this is a decisioningEngine event type, can we exclude the `Mobile` part?

Reply:
They originated from mobile, but we could rename.  what do you recommend?  I think `EVENT_TYPE` is a bit too generic.

Reply:
Since we are in Web and this feature is `in-browser-messages` it should probably be named `IN_APP_EVENT_TYPE` or `BROWSER_MESSAGING_EVENT_TYPE`. Using Mobile is really confusing.

Reply:
yeah, it is confusing.  The challenge here is that it really is a "Mobile" event that we are supporting on the web.  The mobile SDK has an event based model that is utilized for most functionality (AFAIK).  They implemented IAM first, so naturally they leveraged event types that made sense within that context.  On the web we now support a subset of mobile event types and sources when evaluating rulesets.  They aren't really coupled to "in app" or "browser messaging"  So naming it is a bit of a challenge.  That's why i just called it "mobile event type" 😬 . 

The `MOBILE_EVENT_TYPE` and `MOBILE_EVENT_SOURCE` are only used in the decisioning context and vary slightly depending on if `sendEvent` or `evaluateRulesets` is used.  Now that I think about it maybe it could be called `CONTEXT_EVENT_TYPE` and `CONTEXT_EVENT_SOURCE`?  That is a bit more self evident for what it is used for.  Thoughts?